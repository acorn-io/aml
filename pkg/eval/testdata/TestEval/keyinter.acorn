a: 1
"x\(a)": a
y: x1
