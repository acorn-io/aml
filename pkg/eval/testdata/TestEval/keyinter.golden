{
  "a": 1,
  "x1": 1,
  "y": 1
}